targets/PythonSdk/source/source/PlayFabManualTest.py

```diff
@@ -0,0 +1,57 @@
+# idea here is to include PlayFabSettings, set them,
+# include API.py (or PlayFabSErverAPI.py) and see if you can call a function out of it
+import PlayFabSettings
+import PlayFabEntityAPI
+import PlayFabServerAPI
+import PlayFabClientAPI
+import PlayFabManualTestSettings
+
+PlayFabSettings.TitleId = PlayFabManualTestSettings.TitleId
+PlayFabSettings.SdkVersionString = PlayFabManualTestSettings.SdkVersionString
+customId = PlayFabManualTestSettings.customId
+
+request = {}
+
+request["CreateAccount"] = True
+request["CustomId"] = customId
+request["TitleId"] = PlayFabSettings.TitleId
+request["LoginTitlePlayerAccountEntity"] = True
+
+customData = ""
+extraHeaders = {}
```

Comment:
extraHeaders are such an extremely minor edge case that most customers should never see or use them. The SDK should default to None for this var, and usually never be known by the customer.

Reply:
Also with extraHeaders, don't provide or demo it at all here.